docs/guides/08-interactivity-events.md

```diff
@@ -0,0 +1,411 @@
+
+
+### Adding pop-ups
+A very common case is to display pop-ups, little emerging windows with information on the features.
+
+You can build the pop-up yourself if you want, but using `Mapbox GL` allows you to easily reuse [mapboxgl.Popup](https://www.mapbox.com/mapbox-gl-js/api/#popup).
+
+So let's adapt a bit the previous 'featureClick' handler. You're going to add some code inside the current handler:
+```js
+interactivity.on('featureClick', featureEvent => {
+    // ...existing code...
+    // Add more code HERE
+});
+```
+
+First just grab the first feature in the interaction, if exists, with this:
+```js
+const feature = featureEvent.features[0];
+if (!feature) {
+    return;
+}
+```
+
+And then you can create the pop-up with this code:
+```js
+const coords = featureEvent.coordinates;
+const html = `
+    <h1>${feature.variables.name.value}</h1>
+    <p>Population: ${feature.variables.popK.value.toFixed(0)}K</p>
+`;
+new mapboxgl.Popup()
+    .setLngLat([coords.lng, coords.lat])
+    .setHTML(html)
+    .addTo(map);
+```
+> For simplicity, we have created a pop-up just linked to the first feature, but you're free to choose the contents (maybe even a paginated pop-up with several cities and some photos?).
+
+At this point, your map looks like:
+<div class="example-map">
+    <iframe
+        id="guides-interactivity-step-4"
+        src="/developers/carto-vl/examples/maps/guides/interactivity/step-4.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+
+### Interactive-based styling
+Interactivity also can help you to define your styles dynamically.
+
+For example with the next code you'll learn something very useful and common: how to style your features when you interact with them, to give more emphasis to the selected ones.
+
+First you have to set up a listener for the `featureEnter` in the current `Interactivity` object. This listener will change the color and size of the features included in the `features` array.
+```js
+interactivity.on('featureEnter', featureEvent => {
+    featureEvent.features.forEach((feature) => {
+        feature.color.blendTo('rgba(0, 255, 0, 0.8)', 100);
+        feature.width.blendTo(20, 100);
+    });
+});
+```
+> [blendTo](/developers/carto-vl/reference/#expressionblendto) is an expression that allows a smooth transition between two values. In this case, the transition makes the original color turn to red and also increase the size of the symbols.
+
+When the `featureLeave` event is fired you can tell your callback to `reset` the color & size of the features
+
+```js
+interactivity.on('featureLeave', featureEvent => {
+    featureEvent.features.forEach((feature) => {
+        feature.color.reset();
+        feature.width.reset();
+    });
+});
+```
+
+### All together
+
+Congrats! You have finished this guide. The final map should look like this:
```

Comment:
I'm concerned about contractions. We're using them in many places, but here we're not using it: "You have finished" instead of "You've finished". I think it's a consistency problem, not a serious one, but we may need someone to take a look to the writing style we're using for the guides.

Reply:
That's true. In general, I try to keep a quite relaxed and colloquial style